src/components/common/form/FormProvider.tsx

```
@@ -0,0 +1,189 @@
+        const isInvalid = validators.some(
+          (validator) => !validator.validate((value as FormValue).toString()),
+        );
+
+        return {
+          [name]: isInvalid,
+        };
+      })
+      .reduce(
+        (acc, value) => {
+          return {
+            ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)